ee/agent/internal/informer_modules_test.go

```diff
@@ -0,0 +1,103 @@
+package internal_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/formancehq/operator/api/formance.com/v1beta1"
+	"github.com/formancehq/stack/components/agent/internal"
+	"github.com/formancehq/stack/libs/go-libs/collectionutils"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestRestrictStatus(t *testing.T) {
+
```

Comment:
If you want tests to be parallel, you need to set t.Parallel() on the child test AND on the main test